pysoa/server/server.py

```diff
@@ -53,9 +53,10 @@
 
 
 try:
-    import asyncio
+    from pysoa.server.internal.event_loop import AsyncEventLoopThread
+    has_asyncio = True
 except ImportError:
-    asyncio = None
+    has_asyncio = False
```

Comment:
Why not just `AsyncEventLoopThread = None`? I ask because my IDE warns me about this, says that `AsyncEventLoopThread` is undefined, but setting `AsyncEventLoopThread = None` here clears that warning and also obviates any need for `has_asyncio`.